app/controllers/v1/base_controller.rb

```diff
@@ -0,0 +1,5 @@
+module V1
```

Comment:
We prefer `/app/controllers/api/v1/`
```ruby
module Api
  module v1
  end
end
```

Reply:
Yes, I saw it in the other PR after I changed it here. I'll try to change it now.